mlflow/utils/databricks_utils.py

```diff
@@ -335,6 +335,57 @@ def get_workspace_url():
         return None
 
 
+def is_mlflowdbfs_available():
+    if not is_in_databricks_runtime():
+        return False
+
+    # allow users to choose not to use mlflowdbfs via an environment variable
+    disable_mlflowdbfs = os.environ.get("DISABLE_MLFLOWDBFS")
+    if disable_mlflowdbfs:
+        return False
+
+    try:
+        spark_session = _get_active_spark_session()
+    except Exception:
+        return False
+
+    if spark_session is None:
+        return False
+
+    from py4j.protocol import Py4JJavaError
```

Comment:
Py4J import should be moved inside the `try/catch` below; we should also catch `ImportError` just in case.